io.catenax.material_demand/1.0.0/MaterialDemand.ttl

```diff
@@ -0,0 +1,329 @@
+   samm:preferredName "Timestamp"@en ;
+   samm:description "Point in time."@en ;
+   samm:dataType xsd:dateTimeStamp .
+
+:MaterialGlobalAssetIdTrait a samm-c:Trait ;
+   samm:preferredName "Material Global Asset ID Trait"@en ;
+   samm-c:baseCharacteristic :GlobalAssetIdCharacteristic ;
+   samm-c:constraint :MaterialGlobalAssetIdConstraint .
+
+:DemandRateEnumeration a samm-c:Enumeration ;
+   samm:preferredName "Demand Rate Enumeration"@en ;
+   samm:description "The demand rate specifies the time horizon of the the demand.\n- day = time between the start of the specified day till the end of the specified day (ISO-8601).\n- calendar week = time between the start of the specified calendar week till the end of the specified calendar week (ISO-8601-1:2019)."@en ;
+   samm:dataType :DemandRateType ;
+   samm-c:values ( :DemandRateCalendarWeek :DemandRateDay ) .
+
+:DemandSeries a samm:Entity ;
+   samm:preferredName "Demand Series"@en ;
+   samm:description "Encapsulates the demand series related information."@en ;
+   samm:properties ( :customerLocation :demandCategory :demands [ samm:property :expectedSupplierLocation; samm:optional true ] ) .
+
+:GlobalAssetIdCharacteristic a samm:Characteristic ;
+   samm:preferredName "Global Asset ID Characteristic"@en ;
+   samm:description "Defines the data type of a Global Asset ID"@en ;
+   samm:dataType xsd:string .
+
+:MaterialGlobalAssetIdConstraint a samm-c:RegularExpressionConstraint ;
+   samm:preferredName "Material Global Asset ID Constraint"@en ;
+   samm:description "The provided regular expression ensures that the string starts with urn:uuid: followed by the UUID which is composed of five groups of characters separated by hyphens, in the form 8-4-4-4-12 for a total of 36 characters (32 hexadecimal characters and 4 hyphens)."@en ;
+   samm:value "(^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$)|(^urn:uuid:[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$)" .
+
+:DemandRateType a samm:Entity ;
+   samm:preferredName "Demand Rate Type"@en ;
+   samm:description "Type defining the possible demand rates."@en ;
+   samm:properties ( :demandRateCode [ samm:property :demandRateName; samm:notInPayload true ] ) .
+
+:DemandRateCalendarWeek a :DemandRateType ;
+   :demandRateCode "WKS" ;
+   :demandRateName "Calendar Week"@en .
+
+:DemandRateDay a :DemandRateType ;
+   :demandRateCode "DAY" ;
+   :demandRateName "Day"@en .
+
+:customerLocation a samm:Property ;
+   samm:preferredName "Customer Location"@en ;
+   samm:description "The Business Partner Number Site (BPNS) of the site at which the customer needs the specified material for this demand series."@en ;
+   samm:characteristic ext-number:BpnsTrait ;
+   samm:exampleValue "BPNS8888888888XX" .
+
+:demandCategory a samm:Property ;
+   samm:preferredName "Demand Category"@en ;
+   samm:description "Type of demand for this demand series."@en ;
+   samm:characteristic :DemandCategory .
```

Comment:
By rule, property and referenced characteristic should not have same name. could you please provide little different name to either one of these.

Reply:
renamed